plugins/command-line/prism-command-line.js

```diff
@@ -200,3 +212,4 @@
 	});
 
 }());
+// vim: set tabstop=2 shiftwidth=2 noexpandtab:
```

Comment:
?

Reply:
It's a vim modeline (https://vim.fandom.com/wiki/Modeline_magic).  I put it in while editing to ensure tab indents and was in two minds whether to take it out.  Useful for any other vim users.  Happy to remove.

Reply:
We already have a linter in place to ensure proper formatting. No need for any vim magic :)

I also want to say that I think it's a really bad idea to include editor configurations into source code files.